src/persist-client/src/internal/state.rs

```diff
@@ -1107,6 +1107,19 @@ pub enum HollowBlobRef<'a, T> {
     Rollup(&'a HollowRollup),
 }
 
+/// A rollup that is currently being computed.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Arbitrary, Serialize)]
+pub struct ActiveRollup {
```

Comment:
Would you mind stashing the seqno of the in-progress rollup here as well? Seems like useful debug information at a minumum, might be handy later on, and should be cheap...